HSGoogleDrivePicker/HSGoogleDrivePicker/HSDriveManager.m

```diff
@@ -157,9 +157,8 @@ - (GTMOAuth2ViewControllerTouch *)createAuthController {
                       delegate:self
                       finishedSelector:@selector(viewController:finishedWithAuth:error:)];
     
-    UIBarButtonItem *done=[[UIBarButtonItem alloc] initWithBarButtonSystemItem:UIBarButtonSystemItemDone
-                                                                        target:self
-                                                                        action:@selector(cancel:)];
+    UIBarButtonItem *done=[[UIBarButtonItem alloc] initWithTitle:@"Cancel" style:UIBarButtonItemStylePlain target:self action:@selector(cancel:)];
+                           
```

Comment:
I started to try to set the font on a UIBarButtonItem, but I think I would have to access the nav bar to do that.